Sources/App/Core/DocumentationVersion.swift

```diff
@@ -17,17 +17,12 @@ import Foundation
 import Fluent
 import SemanticVersion
 
+struct DocumentationVersionMetadata {
```

Comment:
Is this perhaps just `DocumentationMetadata` with `versions` being an aspect (property) of it?